apps/rtm/controllers/add-report.js

```diff
@@ -13,7 +13,6 @@ module.exports = class AddReportController extends Controller {
     var reports = req.sessionModel.get('report') || [];
```

Comment:
I think it make sense to have a dedicated ticket for doing this, rather than little bits here and there